### PR TITLE
Fixes issue#44

### PR DESCRIPTION
**Pull request by rbark**

### CODE REVIEW

**Comment by stofolus** [comment]
What does the regex change actually do?

**Comment by rbark** [comment]
I added \D* at the end, so any non-digits is allowed, 198511179882L  is alloeed but 1985111798828 is not

**Comment by stofolus** [comment]
It seems to be failing a few test though

**Comment by rbark** [comment]
Yes, I noticed that,  I just run the regexp tests and added test for the issue, and something else broke, I will see if I can find out why, maybe my idea wasn't so  good as I thought.

**Comment by rbark** [comment]
I h8 regexp @ the moment

File: lib/pnr.test.js [hunk 1]
```diff
@@ -69,6 +69,11 @@ describe(`regex`, () => {
   test(`Should match pnrs with a trailing character`, () => {
     expect(Pnr.regex.test(`198511179882L`)).toBeTruthy();
   });
+  test(`Should not match if pnr is inside numberserie`, () => {
```
Comment (hansson) [inline lib/pnr.test.js]: I do not like it when there are two runs of the method under test in the same test case, in this case `Pnr.regex.test`. Maybe split it into one test for "should not match if pnr is inside phone number" and one should not match if pnr is inside serialVersionId"?

Comment (rbark) [reply lib/pnr.test.js]: Fixed that, even if I don't totally agree, there are a lot of testcases with multiple asserts, and these is  testing the same thing, i just took the cases from the raised issue.  But anyway, I have changed it.

Comment (hansson) [reply lib/pnr.test.js]: Great! It's not multiple asserts that I see as a problem, what I see as a problem is multiple runs of the method under test in one test case.

This is fine:
```
const result = myMethod()
expect(result.length).toBe(2);
expect(result.size).toBe(2);
```

This I do not like:
```
expect(myMethod().length).toBe(2);
expect(myMethod().size).toBe(2);
```

**Comment by stofolus** [comment]
Looks good!

I'll merge it this weekend. Might wait with a build until we have a few more things for a bigger release though.

**Pull request merged**